Library/Formula/libforensic1394.rb

```diff
@@ -0,0 +1,29 @@
+class Libforensic1394 < Formula
+  homepage "https://freddie.witherden.org/tools/libforensic1394/"
+  url "https://freddie.witherden.org/tools/libforensic1394/releases/libforensic1394-0.2.tar.gz"
+  sha256 "50a82fe2899aa901104055da2ac00b4c438cf1d0d991f5ec1215d4658414652e"
```

Comment:
Our convention is to use sha1 unless the project in question publishes sha256 hashes.